app/lib/widgets/initial/MuninLoginPage.dart

```diff
@@ -96,7 +96,11 @@ class _MuninLoginPageState extends State<MuninLoginPage> {
         HttpHeaders.userAgentHeader: _userAgent,
       },
     ));
-//    dio.interceptors.add(LogInterceptor(requestBody: true, responseBody: true));
+    // Log for development builds
+    if (Application.environmentValue.environmentType ==
+        EnvironmentType.Development) {
+      dio.interceptors.add(LogInterceptor(requestBody: true, responseBody: true));
+    }
```

Comment:
can we keep this as commented out? iirc I commented it out because request log is spamming the console.

https://flutter.dev/docs/development/tools/devtools/network

Flutter has a powerful Network Inspector and most of the times if I want to debug, I would use this